v2/internal/frontend/assetserver/common.go

```diff
@@ -39,25 +39,7 @@ func newOptions(optionString string) *Options {
 	return result
 }
 
-func injectHTML(input string, html string) ([]byte, error) {
-	splits := strings.Split(input, "</head>")
-	if len(splits) != 2 {
-		return nil, fmt.Errorf("unable to locate a </head> tag in your html")
-	}
-
-	var result bytes.Buffer
-	result.WriteString(splits[0])
-	result.WriteString(html)
-	result.WriteString("</head>")
-	result.WriteString(splits[1])
-	return result.Bytes(), nil
-}
-
-func extractOptions(htmldata []byte) (*Options, error) {
-	doc, err := html.Parse(bytes.NewReader(htmldata))
-	if err != nil {
-		return nil, err
-	}
+func extractOptions(htmlNode *html.Node) (*Options, error) {
```

Comment:
Again, not sure this is needed now. I can't think of a scenario where you'd want to remove the loading of bindings and runtime from the dev server.